awx/ui_next/src/screens/ObservabilityMetrics/ObservabilityMetrics.jsx

```diff
@@ -0,0 +1,239 @@
+import React, { useEffect, useCallback, useState, useRef } from 'react';
+import { withI18n } from '@lingui/react';
+import { t } from '@lingui/macro';
+import {
+  PageSection,
+  Card,
+  CardHeader,
+  CardBody,
+  Toolbar,
+  ToolbarGroup,
+  ToolbarContent,
+  ToolbarItem,
+  Select,
+  SelectOption,
+} from '@patternfly/react-core';
+
+import LineChart from './LineChart';
+import { MetricsAPI, InstancesAPI } from '../../api';
+import useRequest from '../../util/useRequest';
+import ContentEmpty from '../../components/ContentEmpty';
+import ContentError from '../../components/ContentError';
+
+let count = [0];
+
+// hook thats calls api every 3 seconds to get data
+function useInterval(callback, delay, instance, metric) {
+  const savedCallback = useRef();
+  useEffect(() => {
+    savedCallback.current = callback;
+  }, [callback]);
+  useEffect(() => {
+    function tick() {
+      count.push(count.length);
+      if (instance && metric) {
+        savedCallback.current();
+      }
+    }
+
+    const id = setInterval(tick, delay);
+    return () => {
+      clearInterval(id);
+    };
+  }, [callback, delay, instance, metric]);
+  return { count };
+}
+function ObservabilityMetrics({ i18n }) {
```

Comment:
```suggestion
function Metrics({ i18n }) {
```

As far as the code is concerned, I think we should just call this screen "Metrics".